src/app/products/loan-products/loan-product-stepper/loan-product-preview-step/loan-product-preview-step.component.html

```diff
@@ -489,6 +489,8 @@ <h3 class="mat-h3" fxFlexFill>Configurable Terms and Settings</h3>
     <span fxFlex="60%">{{ loanProduct.allowAttributeOverrides.graceOnPrincipalAndInterestPayment ? 'Yes' : 'No' }}</span>
     <span fxFlex="40%">Number of days a loan may be overdue before moving into arrears:</span>
     <span fxFlex="60%">{{ loanProduct.allowAttributeOverrides.graceOnArrearsAgeing ? 'Yes' : 'No' }}</span>
+    <span fxFlex="40%">Delinquency Bucket:</span>
+    <span fxFlex="60%">{{ loanProduct.delinquencyBucketId ? 'Yes' : 'No' }}</span>
```

Comment:
@josehernandezfintecheandomx This is wrong here...  this section is about what attributes can be overridden during the application for Loan.

Reply:
Done! Updated